### ISSUE TITLE
Feature Request Summary

### ISSUE THREAD

**Issue by Tereseus** [Pie0ne/bwa-u4-feature-request#1]
I overviewed your feature technical design and your handler method and input element is correct. You can easily implement this feature as you have pretty much written out the code in plain English already but there could be some clarification on the technical design. From what I can tell you do know how to do the request so maybe that is not required. I should also state that while this is a good feature to introduce it is also only a minor change but it is a good change. If you would like you could also create another feature such as telling the user that their playlist was saved. You could do this by triggering an alert or showing text in the playList component or creating a new component which would have the text.

Overall good job.

**Comment by Pie0ne** [comment]
Thanks for review. I know this is the simplest feature, but I wanted to end course in time. Later when I'll have time I'm going to do next features probably those you mentioned. This is really good way to practice developer skills, regards.
